<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportKeysDialog</class>
 <widget class="QDialog" name="ImportKeysDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Private Key</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="tabImportPrivateKey">
     <attribute name="title">
      <string>&amp;Import Private Key</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_Display">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_1_ImportPrivateKey">
        <item>
         <widget class="QLabel" name="privateKeyWidgetLabel">
          <property name="text">
           <string>Private Key:</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="privateKey">
          <property name="toolTip">
           <string>Private key to import into your wallet</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2_ImportPrivateKey">
        <item>
         <widget class="QLabel" name="privateKeyLabelWidgetLabel">
          <property name="toolTip">
           <string>Label for this private key in your wallet</string>
          </property>
          <property name="text">
           <string>Label (optional):</string>
          </property>
          <property name="buddy">
           <cstring>privateKeyLabel</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="privateKeyLabel">
          <property name="toolTip">
           <string>Label for this private key in your wallet</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="rescanCheckBox">
        <property name="toolTip">
         <string extracomment="Rescan the blockchain on import (disabled when running in pruned mode)."/>
        </property>
        <property name="text">
         <string>Rescan</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="privateKeyImportTextMessage">
        <property name="text">
         <string></string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_ImportPrivateKey">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <layout class="QVBoxLayout" name="verticalLayout_Bottom">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_Bottom"/>
      </item>
      <item>
       <widget class="QLabel" name="overriddenByCommandLineLabel">
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_Buttons">
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="toolTip">
        <string>Reset all key management options to default.</string>
       </property>
       <property name="text">
        <string>&amp;Reset</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>48</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>&amp;Import</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
